src/org/omegat/util/gui/UIDesignManager.java

```diff
@@ -470,20 +476,39 @@ private static boolean isDarkTheme() {
         return background_brightness < foreground_brightness;
     }
 
+    private static void loadColors(final String scheme) {
+        try {
+            ResourcesUtil.getBundleColorProperties(scheme).forEach((k, v) -> {
+                if (v.toString().length() <= 6) {
+                    UIManager.put(k.toString(), new Color(Integer.parseInt(v.toString(), 16)));  // int(rgb)
+                } else {
+                    long val = Long.parseLong(v.toString(), 16);
+                    int a = (int)(val & 0xFF);
+                    int b = (int)(val >> 8 & 0xFF);
+                    int g = (int)(val >> 16 & 0xFF);
+                    int r = (int)(val >> 24 & 0xFF);
+                    UIManager.put(k.toString(), new Color(r, g, b, a));  // hasAlpha
```

Comment:
I didn't see any colors with alpha in the .properties files you added. And the color picker UI in the Preferences window doesn't seem to let you set colors with alpha.

I would prefer to remove this code if we don't have a use case for it.

Reply:
Here is a case to use it, sorry I've missed it.
```
# Shadow and Stroke has alpha
OmegaT.shadow=80808080
OmegaT.stroke=EED20080
```